node/dataSources/catalog.ts

```diff
@@ -80,11 +80,39 @@ export class CatalogDataSource extends IODataSource {
     {metric: 'catalog-brands'}
   )
 
+  public brandSearch = (query: string = '') =>
+    this.get(`/pub/brand/list/${query}`)
+
   public categories = (treeLevel: number) => this.get(
     `/pub/category/tree/${treeLevel}/`,
     {metric: 'catalog-categories'}
   )
 
+  public categorySearch = (query: string = '', parentId: string = '') =>
+    this.get(`/pub/category/list?filter=${query}&parent=${parentId}`)
+
+  public collectionSearch = async (query: string = '') => {
+    const {
+      vtex: { authToken, account, workspace },
+      cookies
+    } = this.context
+    const clientAuth = cookies.get('VtexIdclientAutCookie')
```

Comment:
I don't think private APIs should be in store-graphql

Reply:
Is this data only accessible via private APIs?

Reply:
Why so?